renderer.go

```diff
@@ -110,6 +113,12 @@ func (tr *TextRenderer) Text(val string, tokenType int) {
 
 	tr.lastRenderToken = token
 
+	if tr.UpperCase {
+		if tokenType == KeywordToken || tokenType == SymbolToken || tokenType == IdentifierToken {
```

Comment:
Nice patch! I'm not the owner, just have a suggestion.

I'd expect identifiers (column names) to be kept in lower case in most cases, unless for Oracle.